tests/RetryAnnotationTraitTest.php

```diff
@@ -17,145 +17,139 @@ class RetryAnnotationTraitTest extends TestCase
     {
-        $this->validateRetryAttemptsAnnotation('');
+        $this->parseRetryAttemptsAnnotation('');
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \InvalidArgumentException
      * @expectedExceptionMessage The @retryAttempts annotation must be an integer but got "'foo'"
      */
-    public function testInvalidStringArgumentTypeToRetryAttemptsAnnotation()
+    public function testInvalidStringArgumentTypeToRetryAttemptsAnnotation(): void
     {
-        $this->validateRetryAttemptsAnnotation('foo');
+        $this->parseRetryAttemptsAnnotation('foo');
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \InvalidArgumentException
      * @expectedExceptionMessage The @retryAttempts annotation must be an integer but got "1.2"
      */
-    public function testInvalidFloatArgumentTypeToRetryAttemptsAnnotation()
+    public function testInvalidFloatArgumentTypeToRetryAttemptsAnnotation(): void
     {
-        $this->validateRetryAttemptsAnnotation('1.2');
+        $this->parseRetryAttemptsAnnotation('1.2');
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \InvalidArgumentException
      * @expectedExceptionMessage The @retryAttempts annotation must be 0 or greater but got "-1"
      */
-    public function testNonPositiveIntegerToRetryAttemptsAnnotation()
+    public function testNonPositiveIntegerToRetryAttemptsAnnotation(): void
     {
-        $this->validateRetryAttemptsAnnotation(-1);
+        $this->parseRetryAttemptsAnnotation(-1);
     }
 
-    public function testValidRetryAttemptsAnnotations()
+    public function testValidRetryAttemptsAnnotations(): void
     {
-        $this->assertEquals(0, $this->validateRetryAttemptsAnnotation(0));
-        $this->assertEquals(0, $this->validateRetryAttemptsAnnotation('0'));
-        $this->assertEquals(1, $this->validateRetryAttemptsAnnotation(1));
-        $this->assertEquals(1, $this->validateRetryAttemptsAnnotation('1'));
-        $this->assertEquals(1, $this->validateRetryAttemptsAnnotation(1.0));
-        $this->assertEquals(1, $this->validateRetryAttemptsAnnotation('1.0'));
+        $this->assertEquals(0, $this->parseRetryAttemptsAnnotation('0'));
+        $this->assertEquals(1, $this->parseRetryAttemptsAnnotation('1'));
+        $this->assertEquals(1, $this->parseRetryAttemptsAnnotation('1.0'));
     }
 
-    public function testClassRetryDelaySeconds()
+    public function testClassRetryDelaySeconds(): void
     {
         $this->assertEquals(1, $this->getRetryDelaySecondsAnnotation());
     }
 
     /**
      * @retryDelaySeconds 2
      */
-    public function testMethodRetryDelaySeconds()
+    public function testMethodRetryDelaySeconds(): void
     {
         $this->assertEquals(2, $this->getRetryDelaySecondsAnnotation());
     }
 
     /**
      * @retryDelaySeconds
-     * @expectedException LogicException
+     * @expectedException \InvalidArgumentException
```

Comment:
Why are we adding the forward slash if we include `InvalidArgumentException` in the import?

Reply:
I had a warning in my IDE but I can't tell if it was out of necessity (which in this case would still work because the exception is from the global namespace) or because the annotation doesn't resolve the use statements.

In other words I'm not sure that if you had:

```php
use Acme\FooException

...

@expectedException FooException
```

that it would work fine